ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -535,9 +557,9 @@
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.1.3;
-				PRODUCT_BUNDLE_IDENTIFIER = dtube.go.app;
+				PRODUCT_BUNDLE_IDENTIFIER = dtubego.app;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "match AppStore dtube.go.app";
+				PROVISIONING_PROFILE_SPECIFIER = "";
```

Comment:
Do not update app identifier & prov profiles please.

Reply:
Revert only these lines. Use a text editor (not Xcode)

Reply:
Ah missed those two lines. 
Actually.. I need it that way on my local. But I will be careful when I make PR.